src/components/avatar-builder/EmojisListHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import Animated, { useAnimatedStyle } from 'react-native-reanimated';
+import { fonts } from '@rainbow-me/styles';
+import { useTheme } from '@rainbow-me/theme';
+import { magicMemo } from '@rainbow-me/utils';
+
+interface Props {
+  title: string;
+  showSectionTitles?: boolean;
+  nextCategoryOffset: Animated.SharedValue<number>;
+}
+
+const EmojisListHeader = ({
+  title,
+  showSectionTitles,
+  nextCategoryOffset,
+}: Props) => {
+  const { colors } = useTheme();
+
+  const animatedStyle = useAnimatedStyle(() => ({
+    opacity: nextCategoryOffset.value,
+  }));
+
+  if (showSectionTitles) {
+    return (
+      <Animated.View
+        style={[
+          sx.sectionHeaderWrap,
+          { backgroundColor: colors.white },
+          animatedStyle,
+        ]}
+      >
+        <Text
+          style={[
+            // @ts-expect-error Font weight type is to broad string vs a union of select strings
```

Comment:
I'd revisit this comment and make it maybe longer but more explanatory. Right now I have no idea what is going on.